Alexa.NET/Response/CanfulfillSlot.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Alexa.NET.Response
+{
+    public class CanfulfillSlot
```

Comment:
Minor - but "fulfill" should be "Fulfill" here to be consistent with the spelling everywhere else.

Change `CanfulfillSlot` to `CanFulfillSlot` which will also update `CanFulfillIntent` since it uses this type